src/components/Stepper.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import SwipeableViews from 'react-swipeable-views'
+
+import PercentageLine from 'components/PercentageLine'
+import { BackButtonMobile } from 'components/BackButton'
+
+const styles = {
+  percBackground: {
+    backgroundColor: 'var(--silver)'
+  },
+  percLine: {
+    height: 5,
+    marginBottom: '0.5rem'
+  }
+}
```

Comment:
These styles are not dynamic so they should be extracted in a CSS file, no?

Reply:
Done